Intersect (Core)/Attributes/Attributes.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Enums;
+
+namespace Intersect.Attributes;
+internal class Attributes
+{
+    public class RelatedTable : Attribute
+    {
+        public GameObjectType TableType { get; set; }
+
+        public RelatedTable(GameObjectType db) { TableType = db; }
+    }
+
+    public class RelatedVariableType : Attribute
```

Comment:
Convention dictates that the attribute classes end in `Attribute` and you put `[AttributeUsage(...)]` on them.

In both of these cases the usage would be `Field` (because the enum values are fields for some reason....)

Reply:
They should also be sealed since I see no reason to extend them.